cluster/gce/gci/configure-helper.sh

```diff
@@ -2578,6 +2578,15 @@ EOF
 function override-kubectl {
     echo "overriding kubectl"
     echo "export PATH=${KUBE_HOME}/bin:\$PATH" > /etc/profile.d/kube_env.sh
+    # Add ${KUBE_HOME}/bin into sudoer secure path.
+    local sudo_path=$(sudo env | grep "^PATH=")
+    if [ -n "${sudo_path}" ]; then
+      sudo_path=${sudo_path#"PATH="}
```

Comment:
nitpick: don't need the quotes in `"PATH="` here, just `${sudo_path#PATH=}` is enough and it's what's most commonly seen...

Reply:
Done